docs/DeveloperGuide.md

```diff
@@ -185,30 +185,30 @@ A `Review` contains the following attributes,
     * Pros: Less overhead as fewer objects are created
     * Cons: Prone to error as a Component might not be correctly changed
 
-### Review Adding feature
+### Review adding feature
 
-#### What is Review Adding feature about?
+#### What is Review adding feature about?
 
 The Add Review mechanism is facilitated by `AddressBook`. This feature enhances `AddressBook` by allowing to store not only `Stall`, but also `Review`. This is stored internally as a `UniqueStallList` and `UniqueReviewList`. `Review` requires a `Stall` as `Review` is stored in `Stall`. Additionally, the feature implements the following operations:
 
 * `AddressBook#addReview(Review)` —  Adds the `Review` to `UniqueReviewList`
 
-For the command, the feature extends `command`, and is implemented as such:
+For the command, the feature extends `Command`, and is implemented as such:
 * `radd s/STALL_INDEX d/DATE c/CONTENT r/RATING [t/TAGS]…`
 
-#### Implementation Flow of Review Adding feature
+#### Implementation flow of Review adding feature
 
 Given below is an example usage scenario and how the Review adding mechanism behaves at each step.
 
 Note: FoodWhere comes with preloaded data, and can be started on a fresh state with the `clear` command.
 
-Step 1. The user launches the application for the first time. FoodWhere will be initialized with the preloaded data.
+Step 1. The user launches the application for the first time. FoodWhere will be initialized with the preloaded data. Ensure that there is an existing `Stall` with index 1.
 
 Step 2. The user executes `radd s/1 d/20-09-2022 c/The food was good, the chicken rice was fresh. r/4` command to create a new `Review` for `Stall` with index 1.
 
 ![AddReview](images/AddReview.png)
 
-#### UML Diagram for Adding Review
+#### UML Diagram for adding Review
```

Comment:
I would consider "UML diagram for adding Review" but that means that "diagram" would need to be consistently in lowercase.

Reply:
Fixed in latest commit